api/cmd/transformer/main.go

```diff
@@ -78,67 +82,50 @@ func main() {
 
 	transformerConfig := &spec.StandardTransformerConfig{}
 	if err := jsonpb.UnmarshalString(appConfig.StandardTransformerConfigJSON, transformerConfig); err != nil {
-		logger.Fatal("Unable to parse standard transformer transformerConfig", zap.Error(err))
-	}
-
-	featureSpecs := make([]*spec.FeatureTableMetadata, 0)
-	if appConfig.FeatureTableSpecJsons != "" {
-		featureTableSpecJsons := make([]map[string]interface{}, 0)
-		if err := json.Unmarshal([]byte(appConfig.FeatureTableSpecJsons), &featureTableSpecJsons); err != nil {
-			panic(err)
-		}
-		for _, specJson := range featureTableSpecJsons {
-			s, err := json.Marshal(specJson)
-			if err != nil {
-				panic(err)
-			}
-			featureSpec := &spec.FeatureTableMetadata{}
-			err = jsonpb.UnmarshalString(string(s), featureSpec)
-			if err != nil {
-				return
-			}
-
-			if err != nil {
-				logger.Fatal("Unable to parse standard transformer featureTableSpecs config", zap.Error(err))
-			}
-			featureSpecs = append(featureSpecs, featureSpec)
-		}
+		panic(errors.Wrap(err, "unable to parse standard transformer transformerConfig"))
 	}
 
-	feastOpts := feast.OverwriteFeastOptionsConfig(appConfig.Feast, appConfig.RedisOverwriteConfig, appConfig.BigtableOverwriteConfig)
-	logger.Info("feast options", zap.Any("val", feastOpts))
-
-	feastServingClients, err := feast.InitFeastServingClients(feastOpts, featureSpecs, transformerConfig)
+	featureTableMetadata, err := parseFeatureTableMetadata(appConfig.FeatureTableSpecJsons)
 	if err != nil {
-		logger.Fatal("Unable to initialize Feast Clients", zap.Error(err))
+		panic(err)
 	}
 
-	s := server.New(&appConfig.Server, logger)
-
 	if transformerConfig.TransformerConfig.Feast != nil {
```

Comment:
nit: it's better to have the normal code path unindented (outside else block). E.g.

```go

if edgeCase {
   handleEdgeCase()
   return
}

handleNormalCase()
```